openfisca_us/parameters/irs/ald/misc/employer_share.yaml

```diff
@@ -1,5 +1,8 @@
-description: Employer share of self-employment tax
+description: Employer share of self-employment tax that can be deducted from net earnings.
 values:
   0000-01-01: 0.5
```

Comment:
Just to note I think this is fine for other cases too: finding the actual start date is time-consuming and probably not worth the effort.